### PR TITLE
Changed the vendor name to Youwe.

### PR DESCRIPTION
**Pull request by leonhelmus**
* Updated the testing suite to resolve new errors.
* Added new copyright.
* Added declare strict type to all files.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 =====================
 
-Copyright © `2017` `MediaCT`
+Copyright © `2017` `Youwe`
```
Comment (mbijnsdorp) [inline LICENSE.md]: Shouldn't the year be updated as well?

File: phpcs.xml [hunk 2]
```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0"?>
-<ruleset>
+<ruleset name="PHPCS"
+         xmlns="http://pmd.sf.net/ruleset/1.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd"
+         xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">
+    <description>PHPCS</description>
+    <!--<exclude-pattern>path/to/exclude/*</exclude-pattern>-->
     <rule ref="./vendor/mediact/coding-standard/src/MediaCT"/>
```
Comment (mbijnsdorp) [inline phpcs.xml]: Is this going to be updated at a later stage?

File: phpmd.xml [hunk 3]
```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0"?>
-<ruleset>
+<ruleset name="PHPMD"
+         xmlns="http://pmd.sf.net/ruleset/1.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd"
+         xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">
+    <description>PHPMD</description>
+    <!--<exclude-pattern>path/to/exclude/*</exclude-pattern>-->
     <rule ref="./vendor/mediact/coding-standard/src/MediaCT/phpmd.xml" />
```
Comment (mbijnsdorp) [inline phpmd.xml]: Is this going to be updated at a later stage?

File: grumphp.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+imports:
+  - resource: 'vendor/mediact/testing-suite/config/default/grumphp.yml'
```
Comment (mbijnsdorp) [inline grumphp.yml:2]: Is this going to be updated at a later stage?

File: composer.json [hunk 5]
```diff
@@ -1,47 +1,61 @@
...
-      "Mediact\\FileMapping\\": "src"
-    }
-  },
-  "autoload-dev": {
-    "psr-4": {
-      "Mediact\\FileMapping\\Tests\\": "tests"
+    "require-dev": {
+        "phpunit/phpunit": "@stable",
+        "kint-php/kint": "@stable",
+        "mediact/testing-suite": "@stable",
```
Comment (Epo123) [inline composer.json]: Should update this as well when possible

File: phpcs.xml (hunk 2 above)
Comment (leonhelmus) [reply phpcs.xml]: yes

File: phpmd.xml (hunk 3 above)
Comment (leonhelmus) [reply phpmd.xml]: yes

**Pull request merged**